bin/PerlDDS/Run_Test.pm

```diff
@@ -464,7 +464,132 @@ sub wait_kill {
 
   my $process = $self->{processes}->{process}->{$name}->{process};
 
-  return PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+  my $result = PerlDDS::wait_kill($process, $wait_time, $name, $verbose, $opts);
+
+  # If opts has indicated that user wants to get core dump/signal information
+  # then find the core dump file if any and print its stack trace.
+  if (defined $opts && defined $opts->{dump_ref} && ${$opts->{dump_ref}}) {
+      # Get the core file pattern
+      my $core_pattern_file = "/proc/sys/kernel/core_pattern";
```

Comment:
This is fancier than what I'd imagined (I imagined we'd either enforce a set pattern or put it into an environment variable). I like this better, though, assuming it works as expected. :)